Header header
int16 total_num
int16[] id
float64[] x_pos
float64[] y_pos
float64[] z_pos